src/pages/createOrder.js

```diff
@@ -301,16 +302,21 @@ class CreateOrderPage extends React.Component {
       return false;
     }
 
-    // Check if we're creating a new organization
+    // Check if we're creating a new profile
     if (!this.state.stepProfile.id) {
       this.setState({ submitting: true });
+      if (this.state.stepProfile.name.match(/^anonym/)) {
+        // we should find something better to check for anonymous in multiple langs
+        // also this could arguably move to the backend
+        this.state.stepProfile.slug = `anonymous-${base62(6)}`;
```

Comment:
As highlighted in the comment, it's clearly backend responsibility, let's move that.